fitsio/fitslib.py

```diff
@@ -4244,7 +4271,11 @@ def add_record(self, record_in):
 
             If the input is a card string, convert is implied True
         """
```

Comment:
It appears the doc string is out of date here.

Reply:
yep, thanks